pallets/asset/src/benchmarking.rs

```diff
@@ -575,4 +669,22 @@ benchmarks! {
             None,
         ).unwrap();
     }: _(user.origin, ticker, AssetMetadataKey::Local(AssetMetadataLocalKey(1)))
+
+    base_transfer {
+        // For the worst case, investor uniqueness is enabled and the portfolios are not the
```

Comment:
Minor, but I think we're not using investor uniqueness anymore in this base case as `make_asset` creates non-IU assets now.

Reply:
Just merged in the develop branch. This is fixed now.